content/articles/2017-07-29-tidymodels.Rmd

```diff
@@ -0,0 +1,49 @@
+---
+title: The tidymodels Package
+author: Max Kuhn
+date: '2018-07-29'
+slug: tidymodels-0-0-1
+description: > 
+  tidymodels 0.0.1 is on CRAN.
+categories:
+  - package
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+The `tidymodels` package is now on [CRAN](http://cran.r-project.org/web/packages/tidymodels). Similar its sister package `tidyverse`, it can be used to install and load tidyverse packages related to modeling and analysis. Currently, it installs and attaches [`broom`](https://broom.tidyverse.org/), [`dplyr`](http://dplyr.tidyverse.org), [`ggplot2`](https://ggplot2.tidyverse.org/), [`infer`](http://infer.netlify.com/), [`purrr `](https://purrr.tidyverse.org/), [`recipes`](https://tidymodels.github.io/recipes/), [`rsample`](https://tidymodels.github.io/rsample/), [`tibble`](https://tibble.tidyverse.org/), and [`yardstick`](https://tidymodels.github.io/yardstick/). 
+
+```{r load}
+library(tidymodels)
+```
+
+
+`tidymodels` also contains a burgeoning list of _tagged packages_. These can be used to install sets of packages for specific purposes. For example, if you are in need to additional tidy tools for analyzing text data:
+
+```{r tag}
+tag_attach("text analysis")
+```
+
+These lists will be updated with each version of the package. 
```

Comment:
The tagged package lists (or whatever a good term for the list is) instead of these?

Reply:
Does this get to your question? 

> These tags will be updated with each version of the package as new packages are released.